### PR TITLE
bitget.fetchDepositAddress add network param

### PR DESCRIPTION
**Pull request by samgermain**
```
% bitget fetchDepositAddress USDT '{"network": "TRC20"}'
2023-02-14T23:56:27.678Z
Node.js: v18.4.0
CCXT v2.7.94
bitget.fetchDepositAddress (USDT, [object Object])
2023-02-14T23:56:30.210Z iteration 0 passed in 506 ms

{
  currency: 'USDT',
  address: 'TFiZ4AtffQJiZ4rcg2ovzm1iV8dJudwuBn',
  tag: undefined,
  network: 'TRC20',
  info: {
    coin: 'USDT',
    address: 'TFiZ4AtffQJiZ4rcg2ovzm1iV8dJudwuBn',
    chain: 'TRC20',
    tag: '',
    url: 'https://tronscan.org/#/transaction/'
  }
}
2023-02-14T23:56:30.210Z iteration 1 passed in 506 ms
```

```
% bitget fetchDepositAddress USDT '{"network": "ETH"}'  
2023-02-15T00:00:36.635Z
Node.js: v18.4.0
CCXT v2.7.94
bitget.fetchDepositAddress (USDT, [object Object])
2023-02-15T00:00:39.713Z iteration 0 passed in 490 ms

{
  currency: 'USDT',
  address: '0x207451fc93101b104cb92129bb71f8d731bdbbba',
  tag: undefined,
  network: 'ERC20',
  info: {
    coin: 'USDT',
    address: '0x207451fc93101b104cb92129bb71f8d731bdbbba',
    chain: 'ERC20',
    tag: '',
    url: 'https://etherscan.io/tx/'
  }
}
2023-02-15T00:00:39.713Z iteration 1 passed in 490 ms
```

### CODE REVIEW

**Comment by carlosmiei** [comment]
[🤖 TestsManager 🤖  Id: 63ec218e3631fae457232519]
 ```

 Keys not found for bitget, skipping tests!
```

**Comment by carlosmiei** [comment]
[🤖 TestsManager 🤖  Id: 63ec22263631fae45723251a]
 ```

 Keys not found for bitget, skipping tests!
```

**Review by carlosmiei — APPROVED** [review]

**Comment by carlosmiei** [comment]
[🤖 TestsManager 🤖  Id: 63ecaf823631fae45723251d]
 ```

 Keys not found for bitget, skipping tests!
```

**Comment by carlosmiei** [comment]
[🤖 TestsManager 🤖  Id: 63ecb319df9986ab32bb1c4e]
 ```

Testing { exchanges: ["bitget"], symbol: "all", keys: { '--js': true, '--php': false, '--python': false, '--python-async': true, '--php-async': false }, exchangeSpecificFlags: { '--sandbox': false, '--verbose': false, '--private': false, '--privateOnly': true }, maxConcurrency: 5 } (run-tests.js:274)
[100%] Testing bitget FAIL (testExchange @ run-tests.js:191)

FAILED bitget (JavaScript): (explain @ run-tests.js:209)

TESTING { exchange: 'bitget', symbol: 'all' } 

599 symbols BTC/EUR, BTC/USDT, DOGE/BTC, ETH/BTC, ETH/EUR + more...
SPOT SYMBOL: BTC/USDT
SWAP SYMBOL: BTC/USDT:USDT
[Skipping] bitget signIn  - not supported
Testing bitget fetchBalance ( )
fetching balance...
[Skipping] bitget fetchAccounts  - not supported
[Skipping] bitget fetchTransactionFees  - not supported
Testing bitget fetchTradingFees ( )
Testing bitget fetchStatus ( )
[Skipping] bitget fetchOrders  - not supported
Testing bitget fetchOpenOrders ( BTC/USDT )
Testing bitget fetchClosedOrders ( BTC/USDT )
Testing bitget fetchMyTrades ( BTC/USDT )
[Skipping] bitget fetchLeverageTiers  - not supported
Testing bitget fetchLedger ( BTC )
[Skipping] bitget fetchTransactions  - not supported
Testing bitget fetchDeposits ( BTC )
Testing bitget fetchWithdrawals ( BTC )
[Skipping] bitget fetchBorrowRates  - not supported
[Skipping] bitget fetchBorrowRate  - not supported
[Skipping] bitget fetchBorrowInterest  - not supported
[Skipping] bitget addMargin  - test not available
[Skipping] bitget reduceMargin  - test not available
[Skipping] bitget setMargin  - not supported
[Skipping] bitget setMarginMode  - test not available
[Skipping] bitget setLeverage  - test not available
[Skipping] bitget cancelAllOrders  - test not available
[Skipping] bitget cancelOrder  - test not available
[Skipping] bitget cancelOrders  - test not available
[Skipping] bitget fetchCanceledOrders  - test not available
[Skipping] bitget fetchClosedOrder  - not supported
[Skipping] bitget fetchOpenOrder  - not supported
[Skipping] bitget fetchOrder  - test not available
[Skipping] bitget fetchOrderTrades  - test not available
[Skipping] bitget fetchPosition  - test not available
[Skipping] bitget fetchDeposit  - not supported
[Skipping] bitget createDepositAddress  - not supported
[Skipping] bitget fetchDepositAddress  - test not available
[Skipping] bitget fetchDepositAddresses  - not supported
[Skipping] bitget fetchDepositAddressesByNetwork  - not supported
[Skipping] bitget editOrder  - test not available
[Skipping] bitget fetchBorrowRateHistory  - not supported
[Skipping] bitget fetchBorrowRatesPerSymbol  - not supported
[Skipping] bitget fetchLedgerEntry  - not supported
[Skipping] bitget fetchWithdrawal  - not supported
[Skipping] bitget transfer  - test not available
[Skipping] bitget withdraw  - not supported
Testing bitget fetchCurrencies ( BTC/USDT )
fetched balance items: 558
fetched 0 open orders
fetched 0 trades
fetched 0 deposits, asserting each...
Fetched 0 ledger items
fetched 0 withdrawals, asserting each...
fetched 0 closed orders, testing each
[Skipping] bitget signIn  - not supported
Testing bitget fetchBalance ( )
fetching balance...
[Skipping] bitget fetchAccounts  - not supported
[Skipping] bitget fetchTransactionFees  - not supported
Testing bitget fetchTradingFees ( )
Testing bitget fetchStatus ( )
[Skipping] bitget fetchOrders  - not supported
Testing bitget fetchOpenOrders ( BTC/USDT:USDT )
Testing bitget fetchClosedOrders ( BTC/USDT:USDT )
Testing bitget fetchMyTrades ( BTC/USDT:USDT )
[Skipping] bitget fetchLeverageTiers  - not supported
Testing bitget fetchLedger ( BTC )
[Skipping] bitget fetchTransactions  - not supported
Testing bitget fetchDeposits ( BTC )
Testing bitget fetchWithdrawals ( BTC )
[Skipping] bitget fetchBorrowRates  - not supported
[Skipping] bitget fetchBorrowRate  - not supported
[Skipping] bitget fetchBorrowInterest  - not supported
[Skipping] bitget addMargin  - test not available
[Skipping] bitget reduceMargin  - test not available
[Skipping] bitget setMargin  - not supported
[Skipping] bitget setMarginMode  - test not available
[Skipping] bitget setLeverage  - test not available
[Skipping] bitget cancelAllOrders  - test not available
[Skipping] bitget cancelOrder  - test not available
[Skipping] bitget cancelOrders  - test not available
[Skipping] bitget fetchCanceledOrders  - test not available
[Skipping] bitget fetchClosedOrder  - not supported
[Skipping] bitget fetchOpenOrder  - not supported
[Skipping] bitget fetchOrder  - test not available
[Skipping] bitget fetchOrderTrades  - test not available
[Skipping] bitget fetchPosition  - test not available
[Skipping] bitget fetchDeposit  - not supported
[Skipping] bitget createDepositAddress  - not supported
[Skipping] bitget fetchDepositAddress  - test not available
[Skipping] bitget fetchDepositAddresses  - not supported
[Skipping] bitget fetchDepositAddressesByNetwork  - not supported
[Skipping] bitget editOrder  - test not available
[Skipping] bitget fetchBorrowRateHistory  - not supported
[Skipping] bitget fetchBorrowRatesPerSymbol  - not supported
[Skipping] bitget fetchLedgerEntry  - not supported
[Skipping] bitget fetchWithdrawal  - not supported
[Skipping] bitget transfer  - test not available
[Skipping] bitget withdraw  - not supported
Testing bitget fetchPositions ( [ 'BTC/USDT:USDT' ] )
[Skipping] bitget fetchPositionRisk  - not supported
[Skipping] bitget setPositionMode  - test not available
[Skipping] bitget fetchOpenInterestHistory  - not supported
Testing bitget fetchFundingRateHistory ( BTC/USDT:USDT )
[Skipping] bitget fetchFundingHistory  - test not available
BadSymbol bitget fetchMyTrades() only supports spot markets
fetched balance items: 5
fetched 0 open orders
Fetched 0 ledger items
fetched 0 deposits, asserting each...
fetched 4 positions, asserting each...
fetched 0 withdrawals, asserting each...
fetched all 20 funding rates
AssertionError The expression evaluated to a falsy value:

  assert (typeof fundingRate['fundingRate'] === 'number')

fetched 2 closed orders, testing each
BadSymbol bitget does not have market symbol BTC/USDT:USDT
Error: Failed private tests [swap]: fetchMyTrades, fetchPositions, fetchFundingRateHistory
    at runPrivateTests (/home/ec2-user/rust-tester/ccxt/js/test/test.js:563:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async testExchange (/home/ec2-user/rust-tester/ccxt/js/test/test.js:466:13)
    at async main (/home/ec2-user/rust-tester/ccxt/js/test/test.js:578:5) Error: Failed private tests [swap]: fetchMyTrades, fetchPositions, fetchFundingRateHistory
    at runPrivateTests (/home/ec2-user/rust-tester/ccxt/js/test/test.js:563:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async testExchange (/home/ec2-user/rust-tester/ccxt/js/test/test.js:466:13)
    at async main (/home/ec2-user/rust-tester/ccxt/js/test/test.js:578:5) (explain @ run-tests.js:212)
FAIL ["bitget"] (run-tests.js:290)
All done, 1 failed (run-tests.js:295)


```

File: js/bitget.js [hunk 1]
```diff
@@ -1511,11 +1513,12 @@ module.exports = class bitget extends Exchange {
...
         const currencyId = this.safeString (depositAddress, 'coin');
         const networkId = this.safeString (depositAddress, 'chain');
+        const parsedCurrency = this.safeCurrencyCode (currencyId, currency);
         return {
-            'currency': this.safeCurrencyCode (currencyId, currency),
+            'currency': parsedCurrency,
             'address': this.safeString (depositAddress, 'address'),
             'tag': this.safeString (depositAddress, 'tag'),
-            'network': networkId,
+            'network': this.networkIdToCode (networkId, parsedCurrency),
```
Comment (samgermain) [inline js/bitget.js:1521]: just leave a note to add these changes on the PR, it will stick in my memory better if I make the changes myself

**Pull request merged**